src/OrleansSQLUtils/Storage/DbConnectionFactory.cs

```diff
@@ -13,10 +14,21 @@ internal static class DbConnectionFactory
 
         private static CachedFactory GetFactory(string invariantName)
         {
+            if (String.IsNullOrWhiteSpace(invariantName))
```

Comment:
More of a discussion point, I see the reasoning using `System.String` instead of the alias when referencing the class functions. Maybe there is something to refactor in this regard if we want to be uniform.